lab4/lab4.c

```diff
@@ -0,0 +1,141 @@
+#include <stdio.h>
+#include <string.h>
+// Made by hkctkuy (Ilya Yegorov)
+
+void find_most_common_string(char *most_common_string, int max_length, FILE *file_pointer) {  // Finding the most common string int file
+
+    int max_number = 0, current_number, current_offset, most_common_string_current;
+    char current_string[max_length], actual_string[max_length];
+
+    fseek(file_pointer, 0, SEEK_SET);
+
+    while (fgets(current_string, max_length, file_pointer) != NULL) {
+
+        current_offset = ftell(file_pointer);
+
+        current_number = 0;
+
+        while (fgets(actual_string, max_length, file_pointer) != NULL) {
```

Comment:
int is... = fgets(actual_string, max_length, file_pointer) != NULL; //?
while (is...)
{}